sdk/monitor/opentelemetry-exporter-azuremonitor/tests/test_protocol.py

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import unittest
+
+from opentelemetry.exporter.azuremonitor import protocol
+
+
+class TestProtocol(unittest.TestCase):
+    def test_object(self):
+        data = protocol.BaseObject()
+        self.assertEqual(repr(data), "{}")
+
+    def test_data(self):
+        data = protocol.Data()
+        self.assertIsNone(data.base_data)
+        self.assertIsNone(data.base_type)
+
```

Comment:
we must change these to the pytest like assert statements later.

for examples 
assert data.base_type is None

Reply:
Curious why is that better?

Reply:
not really better - just consistency with other packages - just to clarify, this is a very low priority.

Reply:
We can make this change when fixing lint once is fully working for us